README.rst

```diff
@@ -58,6 +58,15 @@ The *apidoc* extension uses the following configuration values:
 
    **Optional**, defaults to ``api``.
 
+``apidoc_template_dir``
+   A directory containing user-defined templates. Template files in this
+   directory that match the default apidoc templates (``module.rst_t``,
+   ``package.rst_t``, ``toc.rst_t``) will overwrite them. The default templates
+   can be found in ``site-packages/sphinx/templates/apidoc/``. This path is
+   relative to the documentation source directory.
+
+   **Optional**, defaults to ``templates``.
```

Comment:
How does this work if a `templates` directory doesn't exist. Looking at the extension source, the `--templatedir` argument defaults to unset. Shouldn't we do the same?

Reply:
If the ``templates`` directory does not exist, the argument is ignored. I don't see a benefit to defaulting to unset. Considering the ``conf.py`` generated by ``sphinx-quickstart`` includes a definition for ``template_dir`` (https://github.com/sphinx-doc/sphinx/blob/35965903177c6ed9a6afb62ccd33243a746a3fc0/sphinx/templates/quickstart/conf.py_t#L38), I think users are likely to expect that the same default directory will be used for apidoc templates too. This was my expectation coming in. I made this PR because I was unable to include overwrite the apidoc default templates without editing the source in my site-packages. I think defaulting to ``'templates'`` will save users some headaches.